centreon/config/packages/Centreon.yaml

```diff
@@ -307,6 +307,10 @@ services:
             tags: ['configuration.notification.resource.repositories']
         Core\Notification\Application\Repository\NotifiableResourceRequestProviderInterface:
             tags: ['configuration.notification.notifiable.resource.request.provider']
+        Core\ResourceAccess\Domain\Model\DatasetFilter\DatasetFilterTypeInterface:
+            tags: ['resource.access.dataset.filter.types']
+        Core\ResourceAccess\Application\Repository\DatasetRepositoryProviderInterface:
+            tags: ['resource.access.dataset.filter.resource.repository.providers']
```

Comment:
Can you move all framework configurations to a separate file: src/Core/ResourceAccess/Infrastructure/Configuration/symfony.yaml

Don't forget to add the 'src/Core/ResourceAccess' folder to the folder exclusion rule. (see line 38)